recipes/openblas/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+
+# Depending on our platform, shared libraries end with either .so or .dylib
+if [[ `uname` == 'Darwin' ]]; then
+     export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+#    DYLIB_EXT=dylib
+else
+     export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+#    DYLIB_EXT=so
+fi
+
+# Build all CPU targets and allow dynamic configuration
+# Build LAPACK.
+# Enable threading. This can be controlled to a certain number by
+# setting OPENBLAS_NUM_THREADS before loading the library.
+make QUIET_MAKE=1 DYNAMIC_ARCH=1 BINARY=${ARCH} NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 CFLAGS="-Wno-everything"
```

Comment:
So, I haven't needed the tweak mentioned above for handling threading here. Though I have tried this with the latest OpenBLAS. I think the last time I played with this was 0.2.14 and I just dropped it when building 0.2.15.